Prefabs/Energy_Cannon_with_Energy_Ball.prefab

```diff
@@ -187,6 +187,71 @@
                     "$type": "EditorLockComponent",
                     "Id": 7697210517306281973
                 },
+                "Component_[8308919018390390778]": {
+                    "$type": "EditorScriptCanvasComponent",
+                    "Id": 8308919018390390778,
+                    "configuration": {
+                        "sourceHandle": {
+                            "id": "{6EA0260B-0747-51BF-944D-BE4B312FC90C}",
+                            "path": "scriptcanvas/WeaponImpactDecal.scriptcanvas"
+                        },
+                        "sourceName": "WeaponImpactDecal.scriptcanvas",
+                        "propertyOverrides": {
+                            "source": {
+                                "id": "{6EA0260B-0747-51BF-944D-BE4B312FC90C}",
+                                "path": "scriptcanvas/WeaponImpactDecal.scriptcanvas"
+                            },
+                            "variables": [
+                                {
+                                    "Datum": {
+                                        "isOverloadedStorage": false,
+                                        "scriptCanvasType": {
+                                            "m_type": 4,
+                                            "m_azType": "{FC3DA616-174B-48FD-9BFB-BC277132FB47}"
+                                        },
+                                        "isNullPointer": false,
+                                        "$type": "MultiplayerSample::SpawnDecalConfig",
+                                        "value": {
+                                            "LifeTime": 2.0
+                                        }
+                                    },
+                                    "VariableId": {
+                                        "m_id": "{1850E939-4A77-4323-807E-6EFF953D8810}"
+                                    },
+                                    "VariableName": "Weapon Impact Decal Config",
+                                    "InitialValueSource": 1
+                                }
+                            ],
+                            "overrides": [
+                                {
+                                    "Datum": {
+                                        "isOverloadedStorage": false,
+                                        "scriptCanvasType": {
+                                            "m_type": 4,
+                                            "m_azType": "{FC3DA616-174B-48FD-9BFB-BC277132FB47}"
+                                        },
+                                        "isNullPointer": false,
+                                        "$type": "MultiplayerSample::SpawnDecalConfig",
+                                        "value": {
+                                            "Scale": 2.0,
+                                            "Opacity": 0.800000011920929,
```

Comment:
"interesting" opacity choice

Reply:
FWIW, that probably isn't intentional, it's that you can't store exactly 0.8 in a 32-bit float.
https://www.h-schmidt.net/FloatConverter/IEEE754.html